src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationapi/timed/RemoveAccessTokens.kt

```diff
@@ -45,12 +61,11 @@ class RemoveAccessTokens(private val service: RemoveAccessTokensService, private
 @Service
 class RemoveAccessTokensService(private val repository: AuthorizationRepository) {
```

Comment:
Add `@Transactional` annotation at class level if it applies to all functions

Reply:
Doesn't look like you have done this? I'd put a single `@Transactional` annotation at the class level in `RemoveAccessTokensService`. You can then remove the `@Transactional` annotation from `RemoveAccessTokens` (it is not necessary to repeat it and makes more sense at the service level I think)